cmd/docker-machine-driver-kvm/Makefile

```diff
@@ -1,4 +1,4 @@
 default: build
 
 build:
-	GOGC=off go build -i -o docker-machine-driver-kvm
+	GOGC=off go build -tags libvirt.1.2.14 -i -o docker-machine-driver-kvm
```

Comment:
I don't have access to such a machine. However latest Ubuntu LTS (xenial 16.04LTS) has [version 1.3.1 available](http://packages.ubuntu.com/search?keywords=libvirt&searchon=names&suite=xenial-updates&section=all).

Personally I would ignore 14.04.